unfoldingWord/translationCore
# Edit project details from the 3-dot menu in a project card

#### This pull request addresses:
- Edit project details from project card 3-dot menu

#### How to test this pull request:
- Open tC go to the projects list and click the 3-dot menu for a project and you should be able to edit the project information and save it when clicking `continue`.
- Click on the 3-dot menu for another project and this time cancel the edit, which should not be saved.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2726)
